examples/seismic/tutorials/02_rtm.ipynb

```diff
@@ -2,9 +2,7 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
```

Comment:
There should be absolutely 0 changes to this notebook. This needs to be amended, or it will affect our CI

Reply:
I just loaded the latest 02_rtm.ipynb and all the results are cleared.  Before I checked in this last notebook, the notebook had results from a previous GPU run.  Do you want me to check in the notebook with results from a CPU run?

Reply:
I would say just checkout the notebook from master since there shouldn't be any change for it and push and should be gtg (basically this notebook shouldn't show up in the diff).

Reply:
The notebook had a previous run in the results in master.  It seems the results should be removed.  Agree ?  If so, that is the file differences the software is recording.

Reply:
yeah, just remove all edits to 02_rtm, there should be none in this PR